projects/RabbitMQ.Client/client/api/IChannel.cs

```diff
@@ -219,7 +219,7 @@ void BasicPublish<TProperties>(CachedString exchange, CachedString routingKey, i
         ///     Routing key must be shorter than 255 bytes.
         ///   </para>
         /// </remarks>
-        ValueTask BasicPublishAsync<TProperties>(string exchange, string routingKey, in TProperties basicProperties, ReadOnlyMemory<byte> body = default, bool mandatory = false)
+        ValueTask BasicPublishAsync<TProperties>(string exchange, string routingKey, TProperties basicProperties, ReadOnlyMemory<byte> body = default, bool mandatory = false)
```

Comment:
Why changing the in? (This creates a copy for structs, the the property one is a large struct.

Reply:
Can't use ‘in‘ in async methods.